### PR TITLE
feat(ui): allow selecting absolute fee  and utxos to create transaction

### PR DESCRIPTION
**Pull request by gabaldon**

### CODE REVIEW

File: src/store/wallet.js [hunk 1]
```diff
@@ -986,6 +1005,23 @@ export default {
       }
     },
 
+    getUtxoInfo: async function(context) {
+      const request = await context.state.api.getUtxoInfo({
+        wallet_id: context.state.walletId,
+        session_id: context.state.sessionId,
+      })
+      if (request.result) {
+        console.log('result', request.result)
```
Comment (tmpolaczyk) [inline src/store/wallet.js:1014]:
```suggestion
```

File: src/store/wallet.js [hunk 2]
```diff
@@ -672,6 +682,7 @@ export default {
           min_consensus_percentage: parameters.minConsensusPercentage,
         },
       }
+      console.log('create data request', data)
```
Comment (tmpolaczyk) [inline src/store/wallet.js:685]:
```suggestion
```

File: src/components/CreateDataRequestForm.vue [hunk 3]
```diff
@@ -306,13 +355,28 @@ export default {
...
+            dataRequest: this.form.dataRequest,
+            extraCommitRounds: this.form.extraCommitRounds,
+            extraRevealRounds: this.form.extraRevealRounds,
+            fee: this.form.fee,
+            feeType: this.feeType.key,
+            minConsensusPercentage: this.form.minConsensusPercentage,
+            rewardFee: this.form.rewardFee,
+            witnesses: this.form.witnesses,
+            collateral: this.form.collateral,
+          })
```
Comment (Tommytrg) [inline src/components/CreateDataRequestForm.vue:379]:
```suggestion
          this.$emit('create-dr', {
            ...this.form,
            feeType: this.feeType.key,
         })
```

File: src/components/SendValueTransferForm.vue [hunk 4]
```diff
@@ -133,15 +151,18 @@ export default {
         callback()
       }
     }
-
+    // const FEE_TYPE = ['absolute', 'weighted']
```
Comment (Tommytrg) [inline src/components/SendValueTransferForm.vue:154]:
```suggestion
```

File: src/components/SendValueTransferForm.vue [hunk 5]
```diff
@@ -133,15 +151,18 @@ export default {
...
     return {
+      checkedUtxos: [],
+      isIndeterminate: true,
```
Comment (Tommytrg) [inline src/components/SendValueTransferForm.vue:157]: What is `isIndeterminate` for?

**Review by Tommytrg — CHANGES_REQUESTED** [review]

**Comment by gabaldon** [comment]
split in #1717 and #1716

**Pull request closed without merging**